arithmetization/src/main/java/net/consensys/linea/zktracer/module/blake2fmodexpdata/BlakeModexpDataOperation.java

```diff
@@ -123,9 +124,9 @@ private void traceParameter(Trace trace, UnsignedByte stamp) {
   }
 
   private void traceBlakeResult(Trace trace, UnsignedByte stamp) {
-    final Bytes input = computeBlake2fResult();
+    final Bytes hash = computeBlake2fResult();
```

Comment:
Was that a bug ? or just overloading the word `input` ?

Reply:
just renaming. Everything was named input before (same name for callData, hashInput, and hashOutput ...)